map/autoware_lanelet2_map_visualizer/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.14)
+project(autoware_lanelet2_map_visualizer)
+
+find_package(autoware_cmake REQUIRED)
+autoware_package()
+
+find_package(PCL REQUIRED COMPONENTS common io filters)
```

Comment:
Can this be removed ? I do not think the package uses `PCL`.

Reply:
Thank you for pointing out. Yes, it can be removed since this package will not use PCL as you mentioned.
I removed the line in 7810059 .